### ISSUE TITLE
Tooltip is not visible for the “+” (Azure: sign in) button when keyboard focus lands on it in the Azure tree item under browse tab in connections dialogA11y_SQL Azure Data Studio_Azure Tree item_Keyboard

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#26026]
Please do not close this bug. This bug should only be closed by Trusted Tester after veriﬁcation 
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug ﬁxing faster and easier." 

### GitHub Tags:
#A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Nov2024; #SQL Azure Data Studio; #WCAG2.1.1;  #Win32; #Benchmark; #FTP; #STP;  #Keyboard;

### Environment Details:
Application Name: SǪL Azure Data Studio 
Edition: Windows 11 Enterprise 
Version: 24H2 
Installed: 22-10-2024 
OS build: 26100.2033 
Experience: Windows Feature Experience Pack 1000.26100.23.0 
Azure data studio version: 1.49.1

### Steps to Reproduce
1.Launch the application-Azure Database Project and install extension SǪL Database Projects. 
2.Tab to “Connections” tab item present in the left menu and Hit Enter key. 
3.Tab to “New connection” button in the connections section and Hit Enter key. 
4. Tab to “Browse” tab and Hit ENTER key. 
5. Tab to “+” (Azure: sign in) button in the Azure tree item. 
6. Observe whether Tooltip is visible or not when keyboard focus lands on the “+” (Azure: sign in) button. 
7. Similar issue observed in 'new connection' and 'create projects from database' screens

### Actual Results:
Tooltip is not visible for the “+” (Azure: sign in) button when keyboard focus lands on it in the Azure tree item under browse tab in connections dialog. 

### Expected Results:
Tooltip should be visible for the “+” (Azure: sign in) button when keyboard focus lands on it in the Azure tree item under browse tab in connections dialog. 

### 
### User Impact:
Keyboard only users will face difficulty to understand the control if the tooltip is not visible when keyboard focus lands on it. 

### Attachment:
https://github.com/user-attachments/assets/8ae1f9c9-57c5-4518-95e9-a9d725999cda
https://github.com/user-attachments/assets/37e63e3a-41a4-4fef-80cb-d7a31e4a1e16
https://github.com/user-attachments/assets/b4e6c203-a999-499f-9bd4-58ecf68fd222
